core/dbt/include/global_project/macros/materializations/incremental/incremental.sql

```diff
@@ -21,9 +21,17 @@
       {% set backup_relation = existing_relation.incorporate(path={"identifier": backup_identifier}) %}
       {% do adapter.drop_relation(backup_relation) %}
 
+      {% set tmp_identifier = this.identifier ~ "__dbt_tmp" %}
+      {%- set tmp_relation = api.Relation.create(identifier=tmp_identifier,
+                                                          schema=this.schema,
+                                                          database=this.database,
+                                                          type='table') -%} 
+  
+      {% do run_query(create_table_as(False, tmp_relation, sql)) %}
       {% do adapter.rename_relation(target_relation, backup_relation) %}
-      {% set build_sql = create_table_as(False, target_relation, sql) %}
+      {% do adapter.rename_relation(tmp_relation, target_relation) %}
       {% do to_drop.append(backup_relation) %}
+      {% set build_sql = "select 1;" %}
```

Comment:
It's important that the table creation happens as part of statement called `"main"`, so that the correct adapter response is logged to stdout. In this materialization, that "main" statement is going to run whatever is in `build_sql`.

So I think this should instead be:
```suggestion
	{% set build_sql = create_table_as(False, tmp_relation, sql) %}
	{% set need_swap = true %}
```

Then down below, after the main statement finishes (line 46):
```sql
{% if need_swap %}
    {% do adapter.rename_relation(target_relation, backup_relation) %}
    {% do adapter.rename_relation(tmp_relation, target_relation) %}
{% endif %}
```